executor/executor.go

```diff
@@ -370,6 +370,12 @@ func (e *Executor) runNode(node *node.Context) {
 
```

Comment:
i think this will be more readable and understandable for future people if you also call the shutdown under `case <-node.StopCh:`

and remove it from the `stopWorkers` method; then someone looking at this method can see that there are two shutdown cases (normal and forcible) and both call `.Shutdown()`

do you see any problems with this idea?

Reply:
Sure Jeff, I think this should be called during a forcible shutdown as well. Thanks for pointing that out. I will have this changed.